models/tasks.js

```diff
@@ -132,7 +132,9 @@ const fetchPaginatedTasks = async ({
   try {
     let initialQuery = tasksModel;
 
-    if (status === TASK_STATUS.OVERDUE) {
+    if (status === TASK_STATUS.NOT_COMPLETED) {
+      initialQuery = tasksModel.orderBy("status").where("status", "not-in", ["COMPLETED", "AVAILABLE"]);
```

Comment:
Whats the reason for order by status?

Reply:
For some reason "not-in" only works if you order by the specific field.